strings.go

```diff
@@ -306,3 +315,51 @@ func keyStmt(key []byte) *ast.GenDecl {
 		}},
 	}
 }
+
+func strConstBlacklist(node ast.Node, info *types.Info, blacklist map[types.Object]struct{}) {
+	strType := types.Typ[types.String]
+	untypedStr := types.Typ[types.UntypedString]
+
+	constCheck := func(node ast.Node) bool {
+		ident, ok := node.(*ast.Ident)
+		if !ok {
+			return true
+		}
+
+		obj := info.ObjectOf(ident)
+		if obj.Type() == strType || obj.Type() == untypedStr {
```

Comment:
I wonder, is this really necessary? We likely want to start garbling other kinds of literals too (such as integers), so it would be nice to not have this extra restriction. The blacklist map might get a bit bigger, but I think that's OK.

Reply:
Will investigate